website/source/docs/commands/kv/get.html.md

```diff
@@ -46,6 +46,20 @@ Key         Value
 passcode    my-long-passcode
 ```
 
+Return only the "creds" "passcode" key:
+
+```text
+$ vault kv get -field=passcode secret/creds
+my-long-passcode
+```
+  
+Set the Unix environment variable "mySecret" to the "passcode" key:
```

Comment:
I would just remove this example, it's showing the same CLI functionality as above and only works on some systems.